fastq_dl/fastq_dl.py

```diff
@@ -182,37 +187,64 @@ def sra_download(
     se = f"{outdir}/{accession}.fastq.gz"
     pe = f"{outdir}/{accession}_2.fastq.gz"
 
+    # remove existing files if force is selected. 
+    # TODO: only remove if the MD5 checksum is different.
+    if force and Path(se).exists():
+        os.remove(se)
+        logging.warning(f"Overwriting existing files!")
+    if force and Path(pe).exists():
+        os.remove(pe)
+        logging.warning(f"Overwriting existing files!")
+
     if not Path(se).exists() and not Path(pe).exists():
         Path(outdir).mkdir(parents=True, exist_ok=True)
+
+        # TODO: add check of read count as a proxy for checksum?
         outcome = execute(
-            f"fasterq-dump {accession} --split-files --threads {cpus}",
+            f"prefetch {accession} --max-size 10T -o {accession}.sra",
             max_attempts=max_attempts,
             directory=outdir,
             is_sra=True,
         )
+
+        if outcome == SRA_FAILED:
+            return outcome
+        else:
+            outcome = execute(
+                f"fasterq-dump {accession} --split-3 --mem 1G --threads {cpus}",
+                max_attempts=max_attempts,
+                directory=outdir,
+                is_sra=True,
+            )
+
         if outcome == SRA_FAILED:
             return outcome
         else:
             execute(f"pigz --force -p {cpus} -n {accession}*.fastq", directory=outdir)
+            os.remove(f"{outdir}/{accession}.sra")
```

Comment:
```suggestion
            Path(f"{outdir}/{accession}.sra").unlink()
```

This way we can drop the `import os` up top

Reply:
Thanks for the suggestion. I've added a commit that removes the `os` dependency.